### DISCUSSION TITLE
[MPC] Live demo failing to log in

### DISCUSSION THREAD

**Discussion by cooldc314** [Web3Auth/Web3Auth#982] [Help]
I keep getting the following error when trying to log in to the MPC live demo the last couple days at https://demo-mpc.web3auth.io/
[
<img width="1154" alt="Screen Shot 2022-12-02 at 10 20 26 AM" src="https://user-images.githubusercontent.com/8561399/205360002-a253bab5-3f3a-499e-99b8-c279c44a5e9c.png">
](url)

Also, when I was able to log in few days ago, the demo looked updated from the code at https://github.com/torusresearch/w3a-mpc-demo.git Will the demo code at GitHub be updated too?

Thank you.

**Comment by Saviour1001** [comment]
Just tested it out and is working fine for me. Can you check again? Thanks for reporting though.

**Reply by cooldc314** [reply]
<img width="1666" alt="Screen Shot 2022-12-04 at 8 10 36 PM" src="https://user-images.githubusercontent.com/8561399/205547772-0a17a4a2-2ad9-4251-acbe-1deff544b8c8.png">

It's still not working for me.

**Reply by YZhenY** [reply]
can you share where youre based out of?

**Comment by cooldc314** [comment]
US California
